ups/drp_pipe.table

```diff
@@ -15,6 +15,7 @@ setupRequired(faro)
 setupRequired(analysis_drp)
 setupRequired(analysis_tools)
 setupRequired(drp_tasks)
+setupRequired(ap_association)
```

Comment:
Where is this package used?

Reply:
Ah, I got the answer to this from the conversation on Jira.

Reply:
For future reference, here: `ingredients/DRP-minimal-calibration.yaml`.